### PR TITLE
CREATE MODAL POPUP FOR CARD CLICK

### PR DESCRIPTION
**Pull request by peterdtitan**
Stored cards as objects and implemented a modal/popup on click of see projects. Functions were implemented using javascript objects and function calls. The following files were changed: 

- index.html (removed card sections)
- style.css (styled modal)
- script.js (implemented function calls and saved cards as objects)

Screenshot 2022-06-17 at 9 15 16 PM

### CODE REVIEW

**Review by ishimwezachee — COMMENTED** [review]
Hi @peterdtitan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you've worked on alone, while its supposed to be in pair

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -105,73 +104,14 @@ <h2>Multi-Post Stories</h2>
...
-            <ul class="languages">
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>ruby</li>
-            </ul>
-            <a href="https://github.com/peterdtitan/portfolio-build" class="cards-cta" target="blank">See Projects</a>
-        </div>
+    <!--
+    <section>
+
```
Comment (Ismailco) [inline index.html]: - [x] If you are not using some parts of the code kindly remove them, please don't push commented code into your repository, so your code is clean and organized

File: css/style.css [hunk 2]
```diff
@@ -74,6 +74,18 @@ li {
   background-color: #979493;
 }
 
+.modal-cta {
```
Comment (Ismailco) [inline css/style.css:77]: - [x] The popup should follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, margins), kindly use the same font family and font size for the title and the description. also, consider adding the icons for the `Live link button` and `Source button`

|Your design|Figma design|
|-|-|
|Screenshot_20220618_075520|Screenshot_20220618_075541|
|Screenshot_20220618_075408|Screenshot_20220618_075430|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @peterdtitan, @MindPrepDev,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by peterdtitan** [comment]
Updated the styling for the buttons as well as font size and font weight to match the given template.

Screenshot 2022-06-18 at 12 20 11 PM

File: script.js [hunk 3]
```diff
@@ -185,8 +185,8 @@ document.addEventListener('DOMContentLoaded', () => {
                           <div class="modal-description">
                             <div>${project.details}</div>
                             <div class="modal-buttons">
-                              <button class="modal-cta" type="button">See Live</button>
-                              <button class="modal-cta" type="button">See Source</button>
+                              <button class="modal-cta" type="button"><img id="seelive" src='./assets/seelive.png'/></button>
+                              <button class="modal-cta" type="button"><img id="seesource" src='./assets/seesource.png'/></button>
```
Comment (keroloussamy) [inline script.js:189]: - [x] Your project is really good, just one thing to go. Kindly fix this issue on your mobile version, the button is out of the page. To match the Figma design.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @peterdtitan, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

 ## Required Changes ♻️
_Check the comments under the review._





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by peterdtitan** [comment]
Fixed-mobile button issue.

Screenshot 2022-06-18 at 12 42 13 PM

File: script.js (hunk 3 above)
Comment (peterdtitan) [reply script.js:189]: Fixed and updated. 👍🏽

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @peterdtitan, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**